site/docs/components/radio-button/examples.mdx

```diff
@@ -110,11 +110,11 @@ Radio Button supports long text descriptions. If text wraps to multiple lines, a
 
 <LivePreview componentName="radio-button" exampleName="WithFormField" >
 
-### Form Field compatibility
+### Form field compatibility
 
-Radio Button group can be wrapped in a Form Field when it’s displayed within a Form context. This provides functionality built into Form Field for increased accessibility.
+You can wrap `RadioButtonGroup` in a form field when it’s within a form context. This provides functionality built into form field for increased accessibility.
```

Comment:
More of a nitpick, "This provides functionality built into form field for increased accessibility" doesn't really tell me anything